zerver/lib/logging_util.py

```diff
@@ -275,6 +275,7 @@ def format(self, record: logging.LogRecord) -> str:
         client = RequestNotes.get_notes(request).client
         assert client is not None
 
+        assert request.user.is_authenticated
```

Comment:
Hmm, this looks out of place for webhook logging code. I wonder if the well-intentioned b515c2bcbfb178d5ef36040a8740e843ab1efc63 actually broke this?